Library/Formula/cubeb.rb

```diff
@@ -0,0 +1,29 @@
+class Cubeb < Formula
+  desc "Cross-platform audio library."
+  homepage "https://github.com/kinetiknz/cubeb"
+  url "https://github.com/kinetiknz/cubeb/archive/cubeb-0.2.tar.gz"
+  sha256 "cac10876da4fa3b3d2879e0c658d09e8a258734562198301d99c1e8228e66907"
+
+  head "https://github.com/kinetiknz/cubeb.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "autoreconf", "--install"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    test_file = File.open testpath/"main.c", "w"
+    test_file.write "#include <cubeb/cubeb.h>\nint main() { return 0; }"
+    test_file.close
```

Comment:
Cool :) I've added some quick tests that check if the library is working in some fashion, i.e. that it can initialise its runtime context, and then create an audio stream (I don't think there is a good way of verifying that the stream actually works without user feedback).